TeamCode/src/main/java/org/firstinspires/ftc/teamcode/SubSystems/ScoringEndUnit.java

```diff
@@ -10,21 +10,30 @@
 
 public class ScoringEndUnit extends SubsystemBase {
     CuttleServo clawscoringServo;
+    CuttleServo posscoringServo;
```

Comment:
`posscoringServo` did you mean `scoringServo`?

Reply:
i wanted it to be clear it was the position servo and not the claw